src/routes/join/components/SignupPage.js

```diff
@@ -140,7 +134,6 @@ module.exports = class SignupPage extends StrangeForms(React.Component) {
                                     <IconButton
                                         aria-label='toggle password visibility'
                                         onClick={this.handleClickShowPassword}
-                                        onMouseDown={this.handleMouseDownPassword}
```

Comment:
Isn't something like this needed to avoid losing focus on the input?

Reply:
Oh, I think you might be right. I'll fix this.